changelogs/fragments/4816-proxmox-fix-extended-status.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - proxmox inventory - fixed extended status detection for qemu (https://github.com/ansible-collections/community.general/pull/4816).
```

Comment:
```suggestion
  - proxmox inventory plugin - fixed extended status detection for qemu (https://github.com/ansible-collections/community.general/pull/4816).
```

Reply:
Done!

Reply:
Is there a way that I can skip the CI pipeline when making trivial typo fixes like these? :-)

Reply:
Unfortunately no.